lib/jekyll-readme-index/generator.rb

```diff
@@ -1,7 +1,7 @@
 module JekyllReadmeIndex
   class Generator < Jekyll::Generator
     README_REGEX = %r!^/readme(\.[^.]+)?$!i
-    INDEX_REGEX = %r!^/($|index\.(html?|xhtml|xml)$)!i
```

Comment:
modified this to move the fixed part of the index match back as a class-level constant